example/csv/consumer/tests/Service/HttpClientServiceTest.php

```diff
@@ -18,7 +17,7 @@ class HttpClientServiceTest extends TestCase
 {
     public function testGetCsvFile(): void
     {
-        $matcher = new Matcher(new PluginFormatter());
```

Comment:
This looks like a nicer experience for end users (removing an additional import)

Reply:
:hugs: 

There is a small problem with these code: user need to figure out if a plugin is being used or not.